### DISCUSSION TITLE
Bone2d gizmos should scale with zoom in editor

### DISCUSSION THREAD

**Discussion by InTheProcess** [godotengine/godot-proposals#9864] [2D]
The textures on my polygons are small because I am making a pixel art game.  The bones are far too big and cover most of the character.  It is hard to see the animation when the bones are obstructing most of the view.  I suggest that Bone2D gizmos should scale according to the editor's zoom in order to not obstruct the view.  An alternative method would be to add a width slider to scale it manually.  Another feature that could be helpful is to have an option for the bones to automatically hide when an animation is playing.

**Comment by Calinou** [comment]
You can adjust the bones' size in the editor settings (it should be in the **Gizmos > 2D** section).

Adjusting size based on zoom automatically may be a good thing in some scenarios, but it may also be a bad thing in others. I'd probably expect it to be a win in *most* scenarios nonetheless, but I'd expect that some people would prefer bone size to remain constant regardless of zoom level.

**Reply by InTheProcess** [reply]
Then perhaps it should be a toggleable setting.  Maybe it can be in the Gizmos settings.  I think these features will be helpful to many people.

**Reply by InTheProcess** [reply]
Also, I can't find the gizmo settings for 2d.  Only the 3d gizmo settings.  I am using Godot 4.2.2.
